src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/service/SwitchRuleService.java

```diff
@@ -243,6 +405,34 @@ private void rulesResponse(String key, SwitchRulesResponse response) {
         }
     }
 
+    private void rulesResponse(String key, SpeakerCommandResponse response) {
```

Comment:
Please, revise the method name. It's unclear now.

Reply:
Renamed.